Automattic/jetpack
# SEO Tools: make feature available on non-connected sites.

## Proposed changes:

The module was previously restricted to paid Jetpack plans. When we added the options to customize the different SEO options to wp-admin (in the block editor and in Jetpack settings), we made the module available to all plans. It is still restricted to paid plans on WordPress.com though.

As a result:

- On self-hosted, we do not need to be checking if the feature is available; it always is.
- On WordPress.com, we should be checking if the feature is available.
- On self-hosted, the feature can consequently be available on offline mode.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start a brand new JN site.
* Go to Settings > Jetpack constants and enable offline mode.
* Go to Jetpack > Settings > Traffic and enable SEO tools.
* Change some of the settings on that page, and see if they're correctly reflected on your site.
* Go to Posts > Add New
* Open the Jetpack plugin sidebar
    * You should see SEO options there.
* Try changing some of the options and publish your post.
    * The options should be reflected on the frontend.

Now test on WordPress.com simple:

* Sandbox a free WordPress.com site.
* Go to Tools > Marketing > Traffic
    * The SEO options are still not available to you.
* Go to Posts > Add New
    * The SEO options in the sidebar are still not available to you.
* Sandbox a paid WordPress.com site and repeat the steps above.
    * The SEO options are available to you, and making changes should be reflected on the frontend.
